packages/stark-ui/assets/styles/_typography.scss

```diff
@@ -2,30 +2,12 @@
   @return mat-typography-level(nth($list, 1), nth($list, 2), nth($list, 3));
 }
 
-//font-size, line-height, font-weight
-$typography-theme: (
-  font-family: 'Roboto, "Helvetica Neue", sans-serif',
-  display-4: (112px, 112px, 300),
-  display-3: (56px, 56px, 400),
-  display-2: (45px, 48px, 400),
-  display-1: (34px, 40px, 400),
-  headline: (24px, 32px, 400),
-  title: (20px, 32px, 500),
-  subheading-2: (16px, 28px, 400),
-  subheading-1: (15px, 24px, 400),
-  body-2: (14px, 24px, 500),
-  body-1: (14px, 20px, 400),
-  caption: (12px, 20px, 400),
-  button: (map-get($button-theme, font-size), map-get($button-theme, line-height), map-get($button-theme, font-weight)),
-  /* Line-height must be unit-less fraction of the font-size.*/ input: (inherit, 1.125, 400)
-);
-
 @if variable-exists(stark-typography-theme) {
   $typography-theme: map-merge($typography-theme, $stark-typography-theme);
 }
 
 $typography-config: mat-typography-config(
-  $font-family: 'Roboto, "Helvetica Neue", sans-serif',
+  $font-family: map-get($typography-theme, font-family),
   $display-4: mat-typography-level-create(map-get($typography-theme, display-4)),
   $display-3: mat-typography-level-create(map-get($typography-theme, display-3)),
```

Comment:
Could you add on the top of this block a comment with the reference where we can find all these properties?

Reply:
Ok i have created a ticket for that as we will know more about it along the development.

https://github.com/NationalBankBelgium/stark/issues/475

Reply:
:ok: